variables.tf

```
@@ -83,9 +83,9 @@ variable "server_side_encryption_configuration" {
 }
 
 variable "acl" {
-  description = "Bucket ACL"
+  description = "Bucket ACL, new S3 ACL rules affect this resource. Will not create unless explicity set"
```

Keep it simple. Add a commented link to the AWS docs if necessary.

```suggestion
  description = "Canned ACL grant"
```